lib/oauth2.js

```diff
@@ -50,6 +56,10 @@ async function _getNewAccessToken({
   scopes,
   maxRetries = 3
 }) {
+  const cachedAccessToken = accessTokenCache.get(client_id + scopes);
```

Comment:
This will result in a cache miss for every variation of scopes:

```
scopes = ["foo:bar","foo:baz"];
scopes = ["foo:baz","foo:bar"];
```

^^^ cache miss, lexicographically sort the array (at a minimum) if the input is going to be an array)

Reply:
Addressed this here: https://github.com/w3c-ccg/vc-api-test-suite-implementations/pull/42/commits/e78a2c664b0ac554fd463fc02226093e5c26158a

We now have a helper function that produces the cacheKey and uses sort on a copy of scopes.